UKHomeOffice/nmsw-ui
# NMSW-1221 Add 'Show all results' button

# Ticket

NMSW-1221

----

## AC

GIVEN I have applied filters to the View UPT status page
WHEN I click "Show all results"
THEN all passengers are displayed

----

## To test

- run app
- create declaration
- upload a passengers with multiple passengers and statuses (can ask for excel file)
- on UPT page select some filters
- Click show all results
- > Filters should be removed
- > You should see all passengers you uploaded

----

## Notes
